tools/e2etesting/pipeline_orchestrated.yml

```diff
@@ -48,13 +48,32 @@ stages:
   dependsOn: deploy
   condition: or(eq(dependencies.deploy.result, 'Succeeded'), eq(dependencies.deploy.result, 'Skipped'))
   jobs:
-  - job: runtests
-    timeoutInMinutes: 120
-    displayName: 'Run End 2 End Tests'
+  - job: runtestspublisher
+    timeoutInMinutes: 45
```

Comment:
timeout should be at least 60 min, please change for all three test running jobs